pkg/vpcmodel/nodesExplainability.go

```diff
@@ -639,15 +683,33 @@ func (v *VPCConnectivity) getConnection(c *VPCConfig, src, dst Node) (conn *comm
 	return conn, nil
 }
 
-func (rulesInLayers rulesInLayers) string(c *VPCConfig) string {
+// prints either rulesDetails by calling StringDetailsRulesOfFilter or effect of each filter by calling StringFilterEffect
+func (rulesInLayers rulesInLayers) string(c *VPCConfig, isIngress, rulesDetails bool) string {
 	rulesInLayersStr := ""
-	for _, layer := range filterLayers {
+	// order of presentation should be same as order of evaluation:
+	// (1) the SGs attached to the src NIF (2) the outbound rules in the ACL attached to the src NIF's subnet
+	// (3) the inbound rules in the ACL attached to the dst NIF's subnet (4) the SGs attached to the dst NIF.
+	// thus, egress: security group first, ingress: nacl first
+	filterLayersOrder := filterLayers
+	if isIngress {
+		filterLayersOrder[0] = NaclLayer
+		filterLayersOrder[1] = SecurityGroupLayer
+	}
+	for _, layer := range filterLayersOrder {
 		filter := c.getFilterTrafficResourceOfKind(layer)
 		if filter == nil {
 			continue
 		}
 		if rules, ok := rulesInLayers[layer]; ok {
-			rulesInLayersStr += filter.StringRulesOfFilter(rules)
+			if rulesDetails {
+				rulesInLayersStr += filter.StringDetailsRulesOfFilter(rules)
+			} else {
+				thisFilterEffectString := filter.StringFilterEffect(rules)
+				if rulesInLayersStr != "" && thisFilterEffectString != "" {
+					rulesInLayersStr += "; "
```

Comment:
use the const of semicolon
please notice that there is also:
**semicolon      = ";"** 
is vpc model

Reply:
Right, done